x-pack/plugins/security_solution/public/hosts/pages/details/details_tabs.tsx

```diff
@@ -62,9 +61,6 @@ export const HostDetailsTabs = React.memo<HostDetailsTabsProps>(
         <Route path={`${hostDetailsPagePath}/:tabName(${HostsTableType.authentications})`}>
           <AuthenticationsQueryTabBody {...tabProps} />
         </Route>
-        <Route path={`${hostDetailsPagePath}/:tabName(${HostsTableType.hosts})`}>
-          <HostsQueryTabBody {...tabProps} />
-        </Route>
```

Comment:
Unrelated. I found this route with that path (e.g: `security/hosts/name/Host-4dbzugdlqd/allHosts`) that does not exist, Route removed.